prow/jobs_config/knative-extensions/backstage-plugins.yaml

```diff
@@ -28,3 +38,17 @@ jobs:
     requirements: [release, docker]
     excluded_requirements: [gcp]
     max_concurrency: 1
+
+resources: high
+resources_presets:
+  high:
+    limits:
+      memory: 16Gi
+    requests:
+      memory: 12Gi
+
+  default:
+    limits:
+      memory: 10Gi
+    requests:
+      memory: 8Gi
```

Comment:
Do you actually need this memory?

Reply:
Copied from Eventing. I can apply any suggestions you might have.

Reply:
I'd leave the defaults until it becomes a problem?

Reply:
Agreed with @pierDipi here. We're trying to save up on cloud cost, going with common defaults make sense here.